db/migrate/20221026193625_add_user_ref_to_like.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLike < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
+  end
+end
```

Comment:
- [ ]The same here 🤔 Likes belongs-to user through the `'author_id'` column of the Like table. Now, we need to show that, the  `'author_id'` reference the User class modal. Kindly write the query like this so the foreign key name will be `author_id`:

```ruby
add_reference :likes, :author, null: false, foreign_key: { to_table: :users }
```